WordPress/gutenberg
# [Patterns]: Remove bundled patterns

## What?
Resolves: https://github.com/WordPress/gutenberg/issues/46017

The GB bundled patterns have been added to Pattern Directory and tagged as `core` patterns with `6.1` WP version. With that we don't need to bundle them anymore.

## Testing Instructions
1. Ensure that the patterns from this comment: https://github.com/WordPress/gutenberg/issues/46017#issuecomment-1344742302 are loaded properly.
2.  You could just disable Gutenberg in a `6.1` installation. Since we tagged them as `core` patterns with `6.1` version in Pattern directory, they should just show up even without Gutenberg being active.

>- [Header inside full-width background image](https://wordpress.org/patterns/pattern/header-inside-full-width-background-image/)
>- [Simple header with dark background](https://wordpress.org/patterns/pattern/simple-header-with-dark-background/)
>- [Text-only header with tagline](https://wordpress.org/patterns/pattern/text-only-header-with-tagline/)
>- [Simple header with tagline](https://wordpress.org/patterns/pattern/simple-header-with-tagline/)
>- [Site title and menu button](https://wordpress.org/patterns/pattern/site-title-and-menu-button/)
>- [Header with hero image](https://wordpress.org/patterns/pattern/header-with-hero-image/)
>- [Centered header with logo](https://wordpress.org/patterns/pattern/centered-header-with-logo/)
>- [Header with large font size](https://wordpress.org/patterns/pattern/header-with-large-font-size/)
>- [Simple header](https://wordpress.org/patterns/pattern/simple-header/)
>- [Centered footer](https://wordpress.org/patterns/pattern/centered-footer/)
>- [Footer with search, site title, and credit line](https://wordpress.org/patterns/pattern/footer-with-search-site-title-and-credit-line/)
>- [Footer with site title and credit line](https://wordpress.org/patterns/pattern/footer-with-site-title-and-credit-line/)
>- [Footer with navigation and credit line](https://wordpress.org/patterns/pattern/footer-with-navigation-and-credit-line/)
>- [Footer with background color and three columns](https://wordpress.org/patterns/pattern/footer-with-background-color-and-three-columns/)
>- [Centered footer with social links](https://wordpress.org/patterns/pattern/centered-footer-with-social-links/)
>- [Footer with credit line and navigation](https://wordpress.org/patterns/pattern/footer-with-credit-line-and-navigation/)
>- [Left-aligned footer](https://wordpress.org/patterns/pattern/left-aligned-footer/)

## Notes
In my testing it seems I've encountered [this issue](https://github.com/WordPress/pattern-directory/issues/443) that some images fail to load. If that's the case for others as well and not something with my environment maybe we need to update some urls, etc.. in the Pattern Directory patterns?
